content/influxdb/cloud-serverless/write-data/troubleshoot.md

```diff
@@ -49,18 +49,16 @@ InfluxDB uses conventional HTTP status codes to indicate the success or failure
-| `401 "Unauthorized"`            |                                                                         | If the `Authorization` header is missing or malformed or if the [token](/influxdb/cloud-serverless/admin/tokens/) doesn't have [permission](/influxdb/cloud-serverless/admin/tokens/create-token/) to write to the bucket. See [examples using credentials](/influxdb/cloud-serverless/get-started/write/#write-line-protocol-to-influxdb) in write requests. |
-| `404 "Not found"`               | requested **resource type** (for example, "organization" or "bucket"), and **resource name**     | If a requested resource (for example, organization or bucket) wasn't found |
-| `413 “Request too large”`       | cannot read data: points in batch is too large                          | If a request exceeds the maximum [global limit](/influxdb/cloud-serverless/admin/billing/limits/) |
-| `422 "Unprocessable Entity"`    | `message` contains details about the error                              | If the data isn't allowed (for example, falls outside of the bucket's retention period).
-| `429 “Too many requests”`       |                                                                         | If the number of requests exceeds the [adjustable service quota](/influxdb/cloud-serverless/admin/billing/limits/#adjustable-service-quotas). The `Retry-After` header contains the number of seconds to wait before trying the write again. | If a request exceeds your plan's [adjustable service quotas](/influxdb/cloud-serverless/admin/billing/limits/#adjustable-service-quotas)
-| `500 "Internal server error"`   |                                                                         | Default status for an error |
-| `503 "Service unavailable"`     |                                                                         | If the server is temporarily unavailable to accept writes. The `Retry-After` header contains the number of seconds to wait before trying the write again.
+| HTTP response code              | Response body                                                                                                                  | Description    |
+| :-------------------------------| :---------------------------------------------------------------                                                               | :------------- |
+| `204 "No Content"`              | no response body                                                                                                               | If InfluxDB ingested all of the data in the batch |
+| `400 "Bad request"`             | error details about rejected points, up to 100 points: `line` contains the first rejected line, `message` describes rejections | If some or all request data isn't allowed (for example, is malformed or falls outside of the bucket's retention period). The response body indicates if a partial write has occurred or all data has been rejected |
```

Comment:
```suggestion
| `400 "Bad request"`             | error details about rejected points, up to 100 points: `line` contains the first rejected line, `message` describes rejections | If some or all request data isn't allowed (for example, is malformed or falls outside of the bucket's retention period)--the response body indicates whether a partial write has occurred or if all data has been rejected |
```

Reply:
Does `--` render as an em-dash or an en-dash in the docs site? I'm not particularly familiar with what renderer is in use!

Reply:
We render it as as em-dash b/c it's easier to type. It was a friendly point of contention in the past, but laziness won out. I'm not sure if that comes with the renderer or it's a custom style rule.

Reply:
Ah, very cool. I'll store that one in the toolbox 😎